frontend/src/entities/dataset/create/dataset-create.tsx

```diff
@@ -282,17 +289,40 @@ export function DatasetCreate () {
                                 <Select
                                     data-cy='dataset-type-select'
                                     selectedOption={{
-                                        label: state.form.groupName ? state.form.groupName : initCap(state.form.type || ''),
+                                        label: initCap(state.form.type || ''),
```

Comment:
Pretty sure `state.form.type` can never be undefined so we don't need the `|| ''`